src/features/prompts/components/prompt-history.tsx

```diff
@@ -0,0 +1,75 @@
+import { StatusBadge } from "@/src/components/layouts/status-badge";
+import { type NextRouter, useRouter } from "next/router";
+
+type Prompt = {
+  id: string;
+  name: string;
+  version: number;
+  createdAt: Date;
+  isActive: boolean;
+  createdBy: string;
+  creator: string | null;
+};
+
+const PromptHistoryTraceNode = (props: {
+  index: number;
+  prompt: Prompt;
+  currentPromptVersion: number | undefined;
+  setCurrentPromptVersion: (version: number | undefined) => void;
+  router: NextRouter;
+  projectId: string;
+}) => (
+  <div
+    className={`group mb-2 flex cursor-pointer flex-col gap-1 rounded-sm p-2 hover:bg-gray-50 ${
+      props.currentPromptVersion === props.prompt.version ? "bg-gray-100" : ""
+    }`}
+    onClick={() => {
+      props.index === 0
+        ? props.setCurrentPromptVersion(undefined)
+        : props.setCurrentPromptVersion(props.prompt.version);
+    }}
+  >
+    <div // center all the content in the div below horizontally
+      className="flex items-center gap-2"
+    >
+      <span className="rounded-sm bg-gray-200 p-1 text-xs">PROMPT</span>
```

Comment:
The Prompt tag is maybe a bit repeptitive as i see it again. What about putting the Version-X there?

Reply:
Sure, I wasn't in love with it either. My thoughts were to keep it similar to the Trace detail observation tree.